core/dbt/include/README.md

```diff
@@ -1 +1,10 @@
-# Include README
+# Include Module
+
+contains the a few key default `dbt-core` sub directories inlcuding.
+
+- `global_project` which defines the default implementations of jinja2 macros for `dbt-core ` which can be overwritten in each adapter repo to work more in line with those adapter plugins. to view adapter specific jinja2 changes please check the relevant adapter repo.
+
+- `starter_project` which produces the default init project inside the `dbt-core` project after doing the `make dev` command
+
+- A `index.html` file used to generate the docs page after using the `generate docs` command in dbt.
```

Comment:
Can we point out that `index.html` is actually built from the [dbt-docs](https://github.com/dbt-labs/dbt-docs) repo?  The `README` over there has details on how it's built.  And then that file is used when `docs generate` is run.

Reply:
And maybe mention that the index.html is generated by hand prior to a release?

Reply:
I might need a bit more context for that @gshank I haven't had to deal with the dbt-docs so to what degree to we have to regenerate the project? prior to release not sure if we have a write up on that anywhere.

Reply:
Jeremy usually generated the file, but Emily did it last time. The file is created from the dbt-docs repo and copied here.

Reply:
ah, okay interesting, now i'm curious to why we have to do it each release? is it just in case any name or ui changes we make?

Reply:
@gerda is it not generated only when `dbt-docs` changes?  It just needs to be done prior to a release when changes occur since the file gets committed to `dbt-core`.

Reply:
We have sometimes changed attributes in core that dbt-docs depends on. So it needs to be generated for each release, to check that it works.

Reply:
ah that's intriguing, will highlight that and try to have a new version up today.

Reply:
To clarify this point: `dbt-docs` depends on `manifest.json` + `catalog.json`, which are contracted/versioned. If the schema of those artifacts changes (in new minor versions), we always want to test that `index.html` still works / is compatible. We only need to regenerate `index.html` if we've made changes to the `dbt-docs` source code (for compatibility with the new artifact schemas, to fix an unrelated bug, etc).